composer.json

```
@@ -8,5 +8,9 @@
         "dealerdirect/phpcodesniffer-composer-installer": "^0.6.2",
         "wp-coding-standards/wpcs": "^2.2",
         "phpcompatibility/phpcompatibility-wp": "^2.1"
-    }
+    },
+    "scripts": {
+		"phpcs": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcs",
+		"phpcbf": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcbf"
+	}
```

@ will suppress errors

simplify scripts 

```suggestion
    "scripts": {
		"phpcs": "phpcs -s -p",
		"phpcbf": "phpcbf"
	}
```